platform/features/conductor/src/TimeConductor.js

```diff
@@ -0,0 +1,108 @@
+/*****************************************************************************
+ * Open MCT Web, Copyright (c) 2014-2015, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT Web is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT Web includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+/*global define*/
+
+/**
+ * The time conductor bundle adds a global control to the bottom of the
+ * outermost viewing area. This controls both the range for time-based
+ * queries and for time-based displays.
+ *
+ * @namespace platform/features/conductor
+ */
+define(
+    function () {
+        'use strict';
+
+        /**
+         * Wrapper for the `telemetry` capability which adds start/end
+         * times to all requests based on the current state of a time
+         * conductor.
+         *
+         * Note that both start and end times are in units which may
+         * vary depending on the domains of telemetry being used. Most
+         * commonly, these are UNIX timestamps in milliseconds.
+         *
+         * @memberof platform/features/conductor
+         * @constructor
+         * @augments {platform/telemetry.TelemetryCapability}
+         * @param {platform/features/conductor.TimeConductor} timeConductor
+         *        the time conductor which controls these queries
+         * @param {platform/telemetry.TelemetryCapability} telemetryCapability
+         *        the wrapped capability
+         */
+        function TimeConductor(start, end) {
+            this.inner = [ start, end ];
```

Comment:
Yep. Going with `inner.start` and `inner.end` etc. because it's a little easier to add some helpers around (particularly in the related controller & representer)